internal/init/init.go

```diff
@@ -90,38 +91,53 @@ func (c *Client) NewCommand() *cobra.Command {
 			c.UserService = *metalGoClient.UsersApi
 			c.ProjectService = *metalGoClient.ProjectsApi
 
-			include := []string{} // []string | Nested attributes to include. Included objects will return their full attributes. Attribute names can be dotted (up to 3 levels) to included deeply nested objects. (optional)
-			exclude := []string{"devices", "members", "memberships", "invitations", "ssh_keys", "volumes", "backend_transfer_enabled", "updated_at", "customdata", "event_alert_configuration",
-				"timezone", "features", "avatar_url", "avatar_thumb_url", "two_factor_auth", "mailing_address", "max_projects", "verification_stage", "emails", "phone_number", "restricted",
-				"full_name", "email", "social_accounts", "opt_in_updated_at", "opt_in", "first_name", "last_name", "last_login_at"}
-			user, _, err := c.UserService.FindCurrentUser(context.Background()).Include(include).Exclude(exclude).Execute()
```

Comment:
IMO we should stick with loading the current user first so that, if we _were_ given a user token, we avoid loading more projects than necessary or making multiple paginated calls to the projects endpoint.  What is the response and/or error that comes back if we try to run `c.UserService.FindCurrentUser(context.Background()).Include(include).Exclude(exclude).Execute()` with a project token?  Is there something in the response/error that we can use to decide that the token might be a project token?

Reply:
The description of #265 had an approach in mind

Reply:
This PR is already largely following the steps in the issue description.  I still prefer adding project API support in a way that does not add latency to user token initialization, but adding the 2-project limit described in that issue to this PR would at least reduce the added latency.

Reply:
@Chris pointed out that a user might only have 1 project, in which case `metal init` would treat their token as a project token instead of a user token; the token type doesn't have any impact outside of the init command, though, so which endpoint we try to load first is largely a philosophical issue.  I leave the decision up to the contributor as to whether we should remove auto-pagination from `getProjectsOK` or load the current user endpoint first and then try projects if current user 403s.